src/client/components/Comment/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './style.scss';
+import CommentInfo from '../CommentInfo/';
+import CommentInput from '../CommentInput/';
+
+class Comment extends React.Component {
+  constructor(props){
+    super(props);
+    this.CommentInfo = {
```

Comment:
we should use the props instead of a custom object. Now if the component will retrieve the settings via constructor, we will need to change all of the references to the CommentInfo too